specificatie/nationaliteit.yaml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.3
+info:
+  title: Nationaliteit definities
+  version: 2.0.0
+  contact: {}
+paths: {}
+components:
+  schemas:
+    AbstractNationaliteit:
+      type: object
+      properties:
+        type:
+          type: string
+      discriminator:
+        propertyName: type
+        mapping:
+          GbaNationaliteit: '#/components/schemas/GbaNationaliteit'
+          Nationaliteit: '#/components/schemas/Nationaliteit'
+          BehandeldAlsNederlander: '#/components/schemas/BehandeldAlsNederlander'
+          VastgesteldNietNederlander: '#/components/schemas/VastgesteldNietNederlander'
+    Nationaliteit:
+      description: |
+        * **redenOpname** : De reden op grond waarvan de persoon de nationaliteit gekregen heeft.
+      allOf:
+        - $ref: '#/components/schemas/AbstractNationaliteit'
+        - type: object
+          properties:
+            datumIngangGeldigheid:
+              $ref: 'datum.yaml#/components/schemas/AbstractDatum'
+            nationaliteit:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/Waardetabel"
+            redenOpname:
```

Comment:
is er een reden dat redenOpname in elk type wordt herhaald ipv op hoger niveau (abstract nationaliteit) gedefinieerd?

Reply:
idem voor datumIngangGeldigheid

Reply:
redenOpname kan op een hoger niveau gedefinieerd worden als ze voor zowel de GBA als de HC variant gelijk zijn. Voor datumIngangGeldigheid is dit niet het geval